src/utils.js

```diff
@@ -34,3 +34,23 @@ export function useInterval({ callback, delay, runOnStart }) {
     }
   }, [delay, runOnStart]);
 }
+
+export function isFeatureEnabled(feature) {
```

Comment:
What do you think about moving it to his own module/file?

And it would be nice injecting this using CI to allow bypass this in the tests.